xayn-ai/src/coi/system.rs

```diff
@@ -356,7 +358,7 @@ mod tests {
 
     #[test]
     fn test_shift_coi_point() {
-        let coi = PositiveCoi::new(0, arr1(&[1., 1., 1.]).into());
+        let coi = PositiveCoi::new(Uuid::nil().into(), arr1(&[1., 1., 1.]).into());
```

Comment:
i'd suggest to use the mocked uuid here as well
```suggestion
        let coi = PositiveCoi::new(mock_uuid(0).into(), arr1(&[1., 1., 1.]).into());
```